src/shogun/classifier/svm/SVMLight.cpp

```diff
@@ -1605,14 +1605,14 @@ void SVMLight::update_linear_component_mkl(
 						W[j*num_kernels+n]+=(a[i]-a_old[i])*compute_kernel(i,j)*(float64_t)label[i];
 				}
 			}
-			w1[n]=0.0 ;
+			
 		}
 
 		// restore old weights
 		kernel->set_subkernel_weights(SGVector<float64_t>(w_backup,num_weights));
 
 		SG_FREE(w_backup);
-		SG_FREE(w1);
+		
```

Comment:
extra empty line... this should be simply removed

Reply:
I'll correct that